sonar/modules/documents/templates/documents/record.html

```diff
@@ -26,18 +26,32 @@
 
 <section class="mt-3">
   <div class="row">
-    <div class="col-lg-3">
+    <div class="col-lg-3 text-center">
       {% if files and files | length > 0 %}
       <div class="mb-4">
         {{ thumbnail('documents', record, files | first) }}
       </div>
       {% endif %}
+
       <!-- DOCUMENT TYPE -->
       {% if record.documentType %}
-      <h5 class="text-center my-4">{{ _('document_type_' + record.documentType) }}</h5>
+      <h5 class="my-4">{{ _('document_type_' + record.documentType) }}</h5>
+      {% endif %}
+
+      <!-- LICENSE -->
+      {% if record.usageAndAccessPolicy %}
+      <p>
+        <small>
+          {% if record.usageAndAccessPolicy.label %}
```

Comment:
It would be nice to have logos in the future.

Reply:
Thanks for the proposition, I will add a card in the kanban for this.